02-express-tutorial/practiceExpress.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");     // Import express module.
+const path = require("path");           // Import path module.
+
+const app = express();                  // Invoke Express method.
+
+// This allows Express to take care of the status codes, content type, file path, etc automatically for me.
+app.use(express.static("./new-public"));
+
+app.get("/", (req, res) => {
+    res.sendFile(path.resolve(__dirname, "./new-public/index.html"));
+});
```

Comment:
Reminder: this is unnecessary if the static directory has an `index.html` file.

Reply:
Noted Taylor.

I have commented that code block out to keep my code DRY.

Thank you.